factcast-store-pgsql/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -29,9 +29,12 @@ databaseChangeLog:
     - include:
          file: factcast/issue346/dropUniqueConstraintForIdent.sql
          relativeToChangelogFile: true
-    - include:
-         file: factcast/issue601/addSerialForPG11Compatibility.sql
-         relativeToChangelogFile: true
+# No longer necessary:
```

Comment:
Why becomes a DB script unnecessary when updating Spring boot? Something to do with new Postgres driver features?

Reply:
either PG Driver or liquibase (probably liquibase) react differently. this patch is not applicable anymore (liquibase does the right thing on bigserial).